src/helix-ui/styles/components/tables.less

```diff
@@ -140,6 +140,16 @@
     }
   }
 
+  .hxLeft {
+    text-align: left;
+  }
+  .hxCenter {
+    text-align: center;
+  }
+  .hxRight {
+    text-align: right;
+  }
```

Comment:
This is duplication of styles, which is adding unnecessary selector complexity for simple utility/helper classes.

I think a better way of fixing this issue would be to intentionally use `!important` for the alignment helpers (`src/helix-ui/styles/helpers/alignment.less`). From a quick look at Bootstrap and Foundation, they do the same thing for most of their helpers.